client/src/components/Public/Map/AddressDetail/AddressDetail.js

```diff
@@ -21,16 +22,20 @@ type AddressDetailProps = {|
   entities: Array<Entity>,
   addressId: number,
   onClick: (e: Event) => void,
+  width: number,
+  height: number,
 |}
 
-const AddressDetail = ({entities, addressId, onClick}: AddressDetailProps) => (
+const DETAILS_HEADER_HEIGHT = 37
```

Comment:
keep constants in constants.js file

Reply:
Ok, but is it good practice to group unrelated constants (I mean just this case) that belong just to specific component and I cant really imagine this to be shared with the rest off app?